src/livemaker/cli/lmlsb.py

```diff
@@ -150,9 +152,59 @@ def validate(input_file):
                 print(f"  script mismatch, {len(orig_bytes)} {len(new_bytes)}")
 
 
+def _dump_json(lsb, pylm, jsonmode):
+    jsonData = {}
```

Comment:
Please use `snake_case` for variable names so that your changes match the rest of the code base

(as noted in the [contributing docs](https://pylivemaker.readthedocs.io/en/latest/contributing.html#coding-style), the code base generally follows PEP8 Python standards with the exception of code that is intended to directly mirror original LiveMaker engine Pascal naming conventions)